### ISSUE TITLE
Tasks reordered when they are reloaded from disk 

### ISSUE THREAD

**Issue by ZhangYiJiang** [CS2103AUG2016-W10-C4/main#66]
### Steps to reproduce: 

1. Open app, add in a few tasks. Note the ordering. 
2. Close and reopen app. Notice the tasks have been reordered 

### Expected behavior: 

The ordering of tasks should be consistent between each launch of the app. Perhaps we should store the creation time of each tasks to keep the default ordering stable.

**Comment by li-kai** [comment]
Tasks are not reordered...rather, it's more of tasks are getting read in order of how it is first written. FILO, so to speak.

Updates do not trigger rewrites, so order is not moved (unlike the order shown to user).
E.g.
```markdown
| User action     	| Storage order 	| User order 	|
|-----------------	|---------------	|------------	|
| User adds 1,2,3 	| 1,2,3         	| 1,2,3      	|
| User updates 1  	| 1,2,3         	| 2,3,1      	|
| User deletes 1  	| 2,3           	| 2,3        	|
| User adds 1     	| 2,3,1         	| 2,3,1      	|
```

**Comment by ZhangYiJiang** [comment]
Fixed by #127
